support/globalconfig/dns.go

```diff
@@ -9,26 +9,44 @@ import (
 	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
 )
 
-func DNSConfig() *configv1.DNS {
-	return &configv1.DNS{
+func DNSConfig(hcp *hyperv1.HostedControlPlane) *configv1.DNS {
```

Comment:
I don't think we need this change. I'd prefer to leave DNSConfig() as is ... just something that returns an empty manifest.

Reply:
I agree thanks for this I refactored.